lesson-14/package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "lesson-14",
+  "devDependencies": {
+    "@shcherbin/eslint-config-node": "^0.3.0",
+    "@shcherbin/stylelint-config": "^0.2.0",
+    "eslint": "^8",
+    "stylelint": "^16.10.0"
+  },
+  "packageManager": "pnpm@9.12.1+sha512.e5a7e52a4183a02d5931057f7a0dbff9d5e9ce3161e33fa68ae392125b79282a8a8a470a51dfc8a0ed86221442eb2fb57019b0990ed24fab519bf0e1bc5ccfc4",
+  "private": true
+}
```

```suggestion
}

```

в конце package.json тоже обычно пустая строка если генерируешь через npm/pnpm init